src/Trace/Link.php

```diff
@@ -0,0 +1,94 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace;
+
+use Google\Cloud\Core\ArrayTrait;
+
+/**
+ * This plain PHP class represents a Link resource. A pointer from the current
+ * span to another span in the same trace or in a different trace. For example,
+ * this can be used in batching operations, where a single batch handler
+ * processes multiple requests from different traces or when the handler
+ * receives a request from a different project.
+ *
+ * @see https://cloud.google.com/trace/docs/reference/v2/rest/v2/Links#link
```

Comment:
Valid links of this style need to include the link text after the URI, like so:

```
@see <URI> <Text>
```

You're currently missing the text.